client/src/components/FileCard.jsx

```diff
@@ -0,0 +1,121 @@
+import './FileCard.css'
+
+import { Folder, DocumentTxt, DocumentImage, Play } from 'grommet-icons'
+
+const FileCard = ({ name, path, type, size }) => {

```

Comment:
let me suggest you to name it as ItemCard 
coz item could be one of the two types:
- file
- folder